### PR TITLE
Fix typos and encoding errors in copyright headers.

### PR DESCRIPTION
**Pull request by giomasce**

### CODE REVIEW

File: include/boost/spirit/home/x3/support/subcontext.hpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*=============================================================================
     Copyright (c) 2001-2014 Joel de Guzman
-    Copyright (c) 2013 Agustin Berge
+    Copyright (c) 2013 Agustín Bergé
```
Comment (Kojoley) [inline include/boost/spirit/home/x3/support/subcontext.hpp:3]: What is wrong here? It was intentionally ASCIIfied in 658ac180986240c75c2485df88354e83d8027b89.

Comment (giomasce) [reply include/boost/spirit/home/x3/support/subcontext.hpp:3]: Sorry, I missed this point. I originally wrote the patch before that change was committed, and failed to notice it when I rebased it.

Comment (djowel) [reply include/boost/spirit/home/x3/support/subcontext.hpp:3]: Can we fix this now, so we can merge this in?

Comment (giomasce) [reply include/boost/spirit/home/x3/support/subcontext.hpp]: I pushed a fixed version of the commit.

**Comment by djowel** [comment]
Many thanks!

**Pull request merged**